.github/workflows/build-chat.yml

```diff
@@ -23,12 +29,28 @@ jobs:
       - name: Install Additional Dependencies
         # Pulls in a bunch of other packages that arent needed for the rest of the actions
         run: npm install @azure/storage-blob@12.1.1
+      - name: 'Az CLI login'
+        uses: azure/login@v1
+        with:
+            client-id: ${{ secrets.AZURE_CLIENT_ID }}
+            tenant-id: ${{ secrets.AZURE_TENANT_ID }}
+            subscription-id: ${{ secrets.AZURE_SUBSCRIPTION_ID }}
```

Comment:
Just curious, 
Have you considered making them environment secrets for `ci` environment? Or are the set as repo secrets because they are also used outside the `ci` environment somewhere?